handler/area_types.go

```diff
@@ -94,16 +95,30 @@ func (h *AreaTypes) Get(w http.ResponseWriter, r *http.Request) {
 	if res != nil {
 		resp.Count = res.Count
 		resp.TotalCount = res.TotalCount
+
 		for _, edge := range res.Dataset.Variables.Edges {
+
+			i := 0
+			if len(edge.Node.Meta.ONSVariable.GeographyHierarchyOrder) > 0 {
+				i, err = strconv.Atoi(edge.Node.Meta.ONSVariable.GeographyHierarchyOrder)
+				if err != nil {
+					h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
+						err: errors.Wrap(err, "unable to cast geography order field to int"),
```

Comment:
We probably want to return a more user friendly error to the caller (using `msg` field of Error struct)

Reply:
Error field now updated.